Jenkinsfile

```diff
@@ -96,32 +134,34 @@ void checkoutRepo(String repo, String dirName=repo) {
 void checkoutRuntimesRepo() {
     String targetBranch = changeTarget
     String [] versionSplit = targetBranch.split("\\.")
-    if(versionSplit.length == 3 
+    if (versionSplit.length == 3
         && versionSplit[0].isNumber()
         && versionSplit[1].isNumber()
        && versionSplit[2] == 'x') {
         targetBranch = "${Integer.parseInt(versionSplit[0]) - 7}.${versionSplit[1]}.x"
     } else {
         echo "Cannot parse changeTarget as release branch so going further with current value: ${changeTarget}"
-    }
-    dir('kogito-runtimes') {
-        githubscm.checkoutIfExists('kogito-runtimes', changeAuthor, changeBranch, 'kiegroup', targetBranch, true)
+       }
+    dir(kogitoRuntimesRepo) {
+        githubscm.checkoutIfExists(kogitoRuntimesRepo, changeAuthor, changeBranch, 'kiegroup', targetBranch, true)
     }
 }
 
-void mavenCleanInstall(String directory, boolean skipTests = false, List profiles = [], String extraArgs = "") {
-    runMaven("clean install", directory, skipTests, profiles, extraArgs)
+void checkoutQuarkusRepo() {
+    dir(quarkusRepo) {
+        checkout(githubscm.resolveRepository(quarkusRepo, 'quarkusio', getQuarkusBranch(), false))
+    }
 }
 
-void runMaven(String command, String directory, boolean skipTests = false, List profiles = [], String extraArgs = "") {
-    mvnCmd = command
-    if(profiles.size() > 0){
-        mvnCmd += " -P${profiles.join(',')}"
-    }
-    if(extraArgs != ""){
-        mvnCmd += " ${extraArgs}"
-    }
-    dir(directory) {
-        maven.runMavenWithSubmarineSettings(mvnCmd, skipTests)
+MavenCommand getMavenCommand(String directory, boolean addQuarkusVersion=true) {
+    mvnCmd = new MavenCommand(this, ['-fae'])
+                .inDirectory(directory)
+    if (addQuarkusVersion && getQuarkusBranch()) {
+        mvnCmd.withProperty('version.io.quarkus', '999-SNAPSHOT')
```

Comment:
Please confirm my understanding:
- Quarkus will have an LTS branch different than the `master` branch which we already check via the Quarkus ecosystem CI
- this branch keeps the `999-SNAPSHOT` all the time
Is that correct?

Reply:
- Yes, this was started due to possibility to migrate Kogito master to Quarkus 1.12 but we still wanted to be sure it works for 1.11. The check will remain and could be useful in the future.
- Quarkus has the particularity to have the same version on all branches, aka 999-SNAPSHOT

Note that the `Quarkus ecosystem CI` does only check kogito runtimes repository. For the whole Kogito ecosystem to be tested, we have nightly run in Jenkins.
This PR check added is just a precheck if needed against Quarkus LTS, if different from the one used in Kogito runtimes.

Reply:
Note that I am going to update as well the PR template to add the `run LTS check` command